src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let sum = days * 40;
+
+  if (days > 6) {
+    sum = sum - 50;
+  } else {
+    if (days > 2) {
+      sum = sum - 20;
+    }
+  }
```

Comment:
You should create variables for 20 and 50 numbers, now I can't understand their meaning
And check point 3 in checklist (good example of returning value from if statement)